nightwatch.conf.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  src_folders: ['/Users/shashankkumar/Desktop/pro2/step_definitions/searchProductSteps.js', '/Users/shashankkumar/Desktop/pro2/step_definitions/hooks.js'],
```

Comment:
add something like this here
<path to your step definition>, <path to your hooks.js>
instead of giving your path